decidim-budgets/app/queries/decidim/budgets/metrics/budget_followers_metric_measure.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Budgets
+    module Metrics
+      class BudgetFollowersMetricMeasure < Decidim::MetricMeasure
+        # Searches for unique Users following the next objects
+        #  - Budgets
+
+        def valid?
+          super && @resource.is_a?(Decidim::Component)
+        end
+
+        def calculate
+          budgets = Decidim::Budgets::Project.where(component: @resource).joins(:component)
```

Comment:
Why do you need the `joins` here?

Reply:
In case this can be removed, check the rest of the files 😄

Reply:
Well, I'm used to add `joins` to be sure to search for records that are trully related but, yeah, in this scenario is useless.